paper/paper.md

```diff
@@ -35,20 +35,12 @@ bibliography: paper.bib
 
 ## Summary
 
-The advent of commercial cloud services has provided researchers with the benefits of flexible and scalable computing and storage resources.
-However, they present a challenge for organizations that require predictability in their expenditure, as the cost control mechanisms are not always well suited to large numbers of research projects with varied budgetary constraints.
-
-In response, we have developed [RCTab](https://rctab.readthedocs.io/) (**R**esearch **C**omputing **Tab**les), an open-source system for automating the management of [subscriptions](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/organize-resources#management-levels-and-hierarchy) on Azure, Microsoft’s cloud computing platform.
-RCTab automates the mundane management tasks of monitoring usage, alerting stakeholders of impending overspend and stopping subscriptions once they are over-budget.
-This saves administrative time, reduces the risk of excessive spending and provides a rich source of data for usage forecasting.
-
-RCTab is designed to be customisable and extensible, so can be easily adapted to the needs of different organisations.
-It is written in Python and has Infrastructure as Code (IaC) deployment for quick and reliable deployment to Azure.
+The advent of commercial cloud services has provided researchers with the benefits of flexible and scalable computing and storage resources. Unfortunately, cloud providers do not always provide the ability to enforce strict budget controls, allowing users to overspend. This can present serious challenges for adoption in research organisations that need to centrally disseminate cloud resources to researchers with independent budgets. In response, we have developed [RCTab](https://rctab.readthedocs.io/) (**R**esearch **C**omputing **Tab**les), an open-source system for budget control and [subscription](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/organize-resources#management-levels-and-hierarchy) management. RCTab enables organisations to centrally manage cloud resources while enforcing strict budget controls. Organisations can allocate budgets, and RCTab will automatically monitor usage and shut down cloud resources when it is consumed. It enables users to monitor their budget usage via a web interface and email alert system. RCTab is designed to be customisable and extensible, is written in Python and can be easily deployed using Terraform. 
```

Comment:
The only thing that jumped out at me is whether it's explicitly why this exists. We say:
> cost control mechanisms are not always well suited to large numbers of research projects.

but could we be more explicit about why they are not well-suited? My understanding is the key issue they don't provide a mechanism to enforce budgets i.e. we can't automatically shut down a subscription on Azure when a user overspends on a budget. 

I tried to reword it a bit to emphasise that - not sure it's the right wording, but that's the idea I was trying to convey.